smtp_test.go

```diff
@@ -0,0 +1,16 @@
+package glutton
+
+import (
+	"testing"
+)
+
+func TestValidateMail(t *testing.T) {
+
+	if validateMail("MAIL FROM:<gunnerforlife00@gmail.com>") != true {
```

Comment:
I think we should use something more generic
```suggestion
	if validateMail("MAIL FROM:<example@example.com>") != true {
```

Reply:
Yeah definitely I just suggested it for the change I will make the PR tomorrow. I currently don't have my laptop.